### PR TITLE
upgrade absinthe phoenix

### PR DESCRIPTION
**Pull request by ayrat555**
I updated my fork of `absinthe_phoenix`. Now, `absinthe_phoenix` should be updated here.
Note that my PR with default query in `absinthe_plug` was merged in the upstream repo so we can move to the upstream when they release a stable version.

Also, I regenerated `mix.lock` file which removed some unused deps.

## Changelog
- upgrade absinthe phoenix
- regenerate mix.lock

### CODE REVIEW

File: apps/block_scout_web/test/block_scout_web/views/api_docs_view_test.exs [hunk 1]
```diff
@@ -30,6 +30,14 @@ defmodule BlockScoutWeb.ApiDocsViewTest do
   end
 
   describe "api_url/1" do
+    setup do
```
Comment (pasqu4le) [inline apps/block_scout_web/test/block_scout_web/views/api_docs_view_test.exs]: I wonder why did you add this `setup` (and the next one).
It seems that all following `test`s here start by setting the env anyway.

Comment (ayrat555) [reply apps/block_scout_web/test/block_scout_web/views/api_docs_view_test.exs]: @pasqu4le  they override default global endpoint configuration which cause some of the tests to fail
